### PR TITLE
FAQ section

### PR DESCRIPTION
**Pull request by tiagojsag**
I implemented the yet unstyled and undesigned FAQ section. I am reusing the react-safona component from the map for the accordion (the WF had one, there is still no design for this page). The current result is showing very little content, but it's a CSS issue, the data is in the DOM.

I extracted the header from the Blog/BlogDetails as reusable components.

The FAQ is being loaded from a JSON file in the public folder - loading from an API is not on the current roadmap, but this way making that change is as easy as it gets.

@hectoruch take a close look at CoverMain and CoverSub - these are basically the same files you created, but decoupled from the blog, so that they can be reused in other pages. Study this pattern and try to use it in the future.

### CODE REVIEW

File: app/src/actions/faq.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { GET_FAQ_ENTRIES } from '../constants';
+import 'whatwg-fetch';
```
Comment (nerik) [inline app/src/actions/faq.js]: https://github.com/Vizzuality/GlobalFishingWatch/pull/99/files#r73844907

File: app/src/components/blog_detail.jsx [hunk 2]
```diff
@@ -2,7 +2,8 @@ import React, { Component } from 'react';
 import listposts from '../../styles/components/c-list-posts.scss';
 import Header from '../containers/header';
 import Footer from './shared/footer';
-import CoverBlogDetail from './blog/cover_blog_detail';
+import { Link } from 'react-router';
+import CoverSub from './shared/CoverSub';
```
Comment (nerik) [inline app/src/components/blog_detail.jsx]: Not sure what 'CoverSub' means at first read. Maybe rename this?

File: app/src/components/faq.jsx [hunk 3]
```diff
@@ -1,24 +1,63 @@
 import React, { Component } from 'react';
```
Comment (nerik) [inline app/src/components/faq.jsx]: Filename case: Faq.jsx

File: app/styles/components/c-cover-page.scss [hunk 4]
```diff
@@ -79,7 +79,7 @@
     }
   }
 
-  .footer_header_blog {
+  .footer_header {
```
Comment (nerik) [inline app/styles/components/c-cover-page.scss]: Code style (use dashes)

File: app/src/components/blog_detail.jsx (hunk 2 above)
Comment (tiagojsag) [inline app/src/components/blog_detail.jsx]: CoverSecondary maybe?

**Pull request merged**